### ISSUE TITLE
Call of Pripyat - Text is unreadable

### ISSUE THREAD

**Issue by ghost** [Swartz27/xray-16#1750]
The bug described [here](https://github.com/OpenXRay/xray-16/issues/865) is still present on my current portable linux build. Any resolution above 1600x900 triggers the bug and renders all text to white rectangles only. It does not occur when setting the language to russian.

**Comment by ikir83** [comment]
Yes I was searching here to see if I'm alone with this issue and I'm not. Using the macOS port and the text is rectangle white line.

**Comment by Pointing8422** [comment]
I have this problem too with an Spanish game of Call of Pripyat. All the menu has white boxes on it.

**Comment by ikir83** [comment]
> I have this problem too with an Spanish game of Call of Pripyat. All the menu has white boxes on it.

If you set the game to 1080p (editing the configure file or trying it blindly) everything will become readable

**Comment by Pointing8422** [comment]
I put it on 1080p and it still has white boxes, on 1600x900 it became readable, but it looks ugly for the low resolution. For now probably will need to use English version.
